### PR TITLE
Switch to sh for syntax highlighting, reorder sub-menu items

### PR DESCRIPTION
**Pull request by andrea-manzi**
# Summary 

Switch to sh for syntax highlighting, reorder sub-menu items

----

**Related issue :**

### CODE REVIEW

File: content/en/users/datatransfer/api/_index.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ full API details. Here we will provide some examples usage using the Curl client
 
 ### Checking how the server sees the identity of the user
 
-```console
+```sh
 curl --capath /etc/grid-security/certificates -E $X509_USER_PROXY 
 --cacert $X509_USER_PROXY https://fts3-public.cern.ch:8446/whoami
 {
```
Comment (gwarf) [inline content/en/users/datatransfer/api/_index.md:87]:
```suggestion
curl --capath /etc/grid-security/certificates -E $X509_USER_PROXY \
  --cacert $X509_USER_PROXY https://fts3-public.cern.ch:8446/whoami
{
```

File: content/en/users/datatransfer/api/_index.md [hunk 2]
```diff
@@ -155,14 +155,14 @@ curl --capath /etc/grid-security/certificates -E $X509_USER_PROXY \
 
 ### Cancelling a job
 
-```console
+```sh
 curl --capath /etc/grid-security/certificates -E $X509_USER_PROXY --cacert $X509_USER_PROXY \
   https://fts3-pilot.cern.ch:8446/jobs/a40b82b7-1132-459f-a641-f8b49137a713 -X DELETE
```
Comment (gwarf) [inline content/en/users/datatransfer/api/_index.md:160]:
```suggestion
curl --capath /etc/grid-security/certificates -E $X509_USER_PROXY \
  --cacert $X509_USER_PROXY \
   -X DELETE \
  https://fts3-pilot.cern.ch:8446/jobs/a40b82b7-1132-459f-a641-f8b49137a713
```

File: content/en/users/datatransfer/clients/_index.md [hunk 3]
```diff
@@ -232,7 +232,7 @@ If negative, there will be no retries.
 
 #### Example
 
-```console
+```sh
 $ fts-rest-transfer-submit -s https://fts3-public.cern.ch:8446 gsiftp://source.host/file gsiftp://destination.host/file
```
Comment (gwarf) [inline content/en/users/datatransfer/clients/_index.md:236]:
```suggestion
fts-rest-transfer-submit -s https://fts3-public.cern.ch:8446 \
  gsiftp://source.host/file gsiftp://destination.host/file
```

File: content/en/users/datatransfer/clients/_index.md [hunk 4]
```diff
@@ -321,7 +321,7 @@ Options
 ````
 
 #### Example
-```console
+```sh
 $ fts-rest-transfer-cancel -s https://fts3-public.cern.ch:8446
```
Comment (gwarf) [inline content/en/users/datatransfer/clients/_index.md:325]:
```suggestion
fts-rest-transfer-cancel -s https://fts3-public.cern.ch:8446 
```

**Comment by andrea-manzi** [comment]
thanks again!

**Pull request merged**